pybv/io.py

```diff
@@ -151,11 +146,6 @@ def write_brainvision(
     channels with non-voltage units such as °C as is (without scaling). For maximum
     compatibility, all signals should be written as µV.
 
-    When passing a list of dict to `events`, the event ``type`` that can be passed is
-    currently limited to one of ``{"Stimulus", "Response", "Comment"}``. The BrainVision
-    specification itself does not limit event types, and future extensions of ``pybv``
-    may permit additional or even arbitrary event types.
```

Comment:
Fine to remove this, but I would somewhere keep a documentation that "Stimulus", "Response", and "Comment" are the classic types in a BrainVision file that is written by BrainVision Recorder.